src/ontology/cl-edit.owl

```diff
@@ -27138,6 +27138,8 @@ EquivalentClasses(obo:CL_1001005 ObjectIntersectionOf(obo:CL_1000892 ObjectSomeV
 
 # Class: obo:CL_1001006 (kidney afferent arteriole cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16990493") Annotation(oboInOwl:hasDbXref "PMID:22733355") Annotation(oboInOwl:hasDbXref "PMID:24066938") Annotation(oboInOwl:hasDbXref "PMID:25834230") obo:IAO_0000115 obo:CL_1001006 "A cell typically found in renal vasculature which is primarily responsible for maintaining renal blood flow and glomerular filtration rate. This cell type regulates blood pressure and fluid balance by controlling the expansion or contraction of the arteriole before the glomerulus through autoregulation mechanisms such as the myogenic response and tubuloglomerular feedback.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16990493") Annotation(oboInOwl:hasDbXref "PMID:22733355") Annotation(oboInOwl:hasDbXref "PMID:24066938") Annotation(oboInOwl:hasDbXref "PMID:25834230") dc:description obo:CL_1001006 "Additionally, these cells interact with renin-containing juxtaglomerular cells to initiate the renin-angiotensin-aldosterone system, which increases blood volume and pressure.")
```

Comment:
QC is failing because of the use of the `dc:description`. Shouldn't this also be in the definition?

```bash
FAIL Rule ../sparql/illegal-annotation-property-violation.sparql: 1 violation(s)
term,annotation
http://purl.obolibrary.org/obo/CL_1001006,http://purl.org/dc/elements/1.1/description
```

Reply:
Ahh - OK.  We need a place for extended descriptions and have talked about using dc:Description but need to owrk on official guidance and (it seems) on QC tests first. I'll put it on the agenda for Monday's call.

Reply:
Could you not use `comment`? In any case, please prefer the `terms` annotations instead of `dc`.

Reply:
We have been using comment for this so far., but the comment field is quite overloaded. We will need something more easily separable for work with CxG.  I suspect DC elements used in place of terms because that's the on available in Protege right now (probably through imports.

Reply:
Is it okay to use comments until the dc:description is fixed on the QC?

Reply:
> Is it okay to use comments until the dc:description is fixed on the QC?
Yes please.